doc/source/whatsnew/v1.2.0.rst

```diff
@@ -766,6 +766,7 @@ Other
 - Bug in :meth:`Index.union` behaving differently depending on whether operand is an :class:`Index` or other list-like (:issue:`36384`)
 - Passing an array with 2 or more dimensions to the :class:`Series` constructor now raises the more specific ``ValueError`` rather than a bare ``Exception`` (:issue:`35744`)
 - Bug in ``dir`` where ``dir(obj)`` wouldn't show attributes defined on the instance for pandas objects (:issue:`37173`)
+- Only set ``-Werror`` as a compiler flag in the CI jobs (:issue:`33315`, :issue:`33314`)
```

Comment:
ok happy to backport, can you move to 1.1.5 whatnew

Reply:
if you'd move the note, pls ping on green (might be a while, merged a lot of things today :->)